files/image_config/platform/rc.local

```diff
@@ -148,6 +148,7 @@ if [ ! -e /host/machine.conf ]; then
     fi
 
     # Extract the previous NOS's partition that contains the migration artifacts
+    . /usr/share/sonic/device/$platform/installer.conf
```

Comment:
I am not sure if this sourcing has meaning here. I thought the /proc/cmdline will be changed during image build and respect ONIE_PLATFORM_EXTRA_CMDLINE_LINUX in installer.conf